src/Filesystem.cpp

```diff
@@ -108,6 +108,24 @@ void Filesystem::mount(const std::string& path) const
 }
 
 
+/**
+ * Mount a folder with write access (includes read access)
+ *
+ * \param path	File path to add.
+ */
+void Filesystem::mountWrite(const std::string& path) const
```

Comment:
This could be better named as `mountReadWrite` and another function could be created: `setWriteDirectory` that just calls `PHYSFS_setWriteDir`.

Reply:
Hmm, this makes me think the existing method should be renamed for symmetry. Perhaps:
- `mountRead`
- `mountReadWrite`

I want to avoid `setWriteDirectory`, since it implies a single write directory. I think we might go more general than that in the future.

Reply:
Yeah, adding a `mountRead` that delegates to `mount` would be useful for understandability. You don't *have* to rename `mount` itself. It's up to you if you want to break the API.

Reply:
Hmm, good point. I sort of want to avoid breaking compatibility in this PR. I'd prefer it to be focused on adding a new feature.

I'm hesitant to add alias method names at this point. Maybe I should just focus on the new method, with the intention of eventually renaming the old method.

One potential future design path is to cleanup the `Game` class, so it is usable by OPHD. The `Game` class initializes the `Filesystem`, so that could bring the `Filesystem` calls back into the library, and eliminate further downstream changes.